app/controllers/projects_controller.rb

```diff
@@ -26,8 +28,20 @@ def edit
 
   private
     def project_params
-      params.require(:project)
-      .permit(:title, :description, :image_url, :target_amount,:completion_date)
```

Comment:
Able to clean this up due to the fact that the dropdown menu sent category_id and country_id through in the project params.